index.html

```diff
@@ -54,5 +54,147 @@ <h1 id="greeting">I’m Andrew. Glad to see you!</h1>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="languages">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+        <div class="projectBTN"><button>See Project</button></div>
+      </li>
+      
+      <li class="card">
+        <img class="card_image" src="Snapshoot Portfolio3.png" alt="" />
+        <p class="projectName_holder">Tonic</p>
+        <ul class="canopy">
+          <li>CANOPY</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+        <p class="service">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="languages">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+        <div class="projectBTN"><button>See Project</button></div>
+      </li>
+      
+      <li class="card">
+        <img class="card_image" src="Snapshoot Portfolio4.png" alt="" />
+        <p class="projectName_holder">Multi-Post Stories</p>
+        <ul class="canopy">
+          <li>CANOPY</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+        <p class="service">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="languages">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+        <div class="projectBTN"><button>See Project</button></div>
+      </li>
+      
+    </ul>
+      <section id="about">
+     <header class="aboutMyself"> 
+     About Myself    
+     </header>
+     <p class="intro">
+      I’m a software developer! I can help you build a product , feature or
+      website Look through some of my work and experience! If you like what
+      you see and have a project you need coded, don’t hesitate to contact me.
+    </p>
+    <p id="connect">LET'S CONNECT</p>
+      <ul class="social">
+        <li>
+          <a href="https://www.twitter.com" target="_self"
+            ><img src="tw.png" alt="twitter icon"
+          /></a>
+        </li>
+        <li>
+          <a href="https://www.linkedin.com" target="_self"
+            ><img src="linkedin.png" alt="linkedin icon"
+          /></a>
+        </li>
+        <li>
+          <a href="https://www.medium.com" target="_self"
+            ><img src="medium.png" alt="medium icon"
+          /></a>
+        </li>
+        <li>
+          <a href="https://www.github.com" target="_self"
+            ><img src="github.png" alt="github icon"
+          /></a>
+        </li>
+        <li>
+          <a href="https://www.vectorstock.com" target="_self"
+            ><img src="v.png" alt="vector stock icon"
+          /></a>
+        </li>
+      </ul>
+      <div class="projectBTN"><button>Get my resume</button></div>
```

Comment:
- [ ] As i have suggested above, you should implement the button interactions here too. :smile: